src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission.go

```diff
@@ -38,31 +45,41 @@ leader handles the request.
 			cl, err := admin.NewClient(fs, cfg)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
-			brokers, err := cl.Brokers(cmd.Context())
-			out.MaybeDie(err, "unable to get broker list: %v", err)
+			if !force {
+				brokers, err := cl.Brokers(cmd.Context())
+				out.MaybeDie(err, "unable to get broker list: %v; to bypass the node version check re-run this with --force", err)
 
-			var b admin.Broker
-			var found bool
-			for _, br := range brokers {
-				if br.NodeID == broker {
-					b, found = br, true
-					break
-				}
-			}
-			if !found {
-				out.Die("unable to find broker %v in the cluster", broker)
-			}
+				var (
+					b             admin.Broker
+					found, anyOld bool
+				)
+				for _, br := range brokers {
+					if br.NodeID == broker {
+						version, err := redpanda.VersionFromString(br.Version)
+						out.MaybeDie(err, "unable to get broker %q version: %v; to bypass the node version check re-run this with --force", br.NodeID, err)
+						isOld := version.Less(redpanda.Version{Year: 23, Feature: 1, Patch: 1})
 
-			version, err := redpanda.VersionFromString(b.Version)
-			out.MaybeDie(err, "unable to get broker version: %v", err)
+						anyOld = anyOld || isOld
+						b, found = br, true
+						break
+					}
+				}
+				if !found {
+					out.Die("unable to find broker %v in the cluster; to bypass the node version check re-run this with --force", broker)
+				}
 
-			if version.Less(redpanda.Version{Year: 23, Feature: 1, Patch: 1}) {
-				// Old brokers (< v22.1) don't have maintenance mode, so we must
-				// check if b.Maintenance is not nil.
-				if b.Maintenance != nil && b.Maintenance.Draining {
-					out.Die(`Node cannot be decommissioned while it is in maintenance mode.
+				// If any of the brokers is older than v23.1.1 we need to check
+				// that the broker that is about to be decommissioned is not
+				// in maintenance mode.
+				if anyOld {
+					// Old brokers (< v22.1) don't have maintenance mode, so we must
+					// check if b.Maintenance is not nil.
+					if b.Maintenance != nil && b.Maintenance.Draining {
+						out.Die(`Node cannot be decommissioned while it is in maintenance mode.
 Take the node out of maintenance mode first by running: 
-    rpk cluster maintenance disable %v`, broker)
+    rpk cluster maintenance disable %v
+to bypass the node version check re-run this with --force`, broker)
```

Comment:
I think we should reference our long help text actually as well, which explains why we have --force. Wdyt about 

```
Read this command's help text for how to bypass the node version check.
```

or

```
To bypass the node version check re-run this with --force; see this command's
help text for more details on why.
```